plugin/pkg/transport/ztransport.go

```diff
@@ -0,0 +1,22 @@
+// generated by servers_generate.go; DO NOT EDIT
+
+package transport
+
+// These transports are supported by CoreDNS.
+const (
```

Comment:
what's the point of these if they are generated anyway?

Reply:
Do you mean maintaining the constants?  I was trying to maintain compatibility as much as possible.